### PR TITLE
Add `n_unsplit_alleles` to gnomAD v4 subset script

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: gnomad_qc/v4/subset.py [hunk 1]
```diff
@@ -168,6 +166,19 @@ def main(args):
             metadata=HEADER_DICT,
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/subset.py:166]: Not part of this PR but since were editting it, wondering if it makes sense to add the TODO num-vcf-shards with `parallel` option here to shard the VCF? If needs more thought -- totally get skipping it but seems like a quick todo? Although we'd probably want a tabix option too so maybe this is just another feature request/PR.

**Review by mike-w-wilson — APPROVED** [review]
LGTM! One note on sharding that is not necessarily tied to this PR. If you don't think its worth tackling now, I'm good merging this.

File: gnomad_qc/v4/subset.py (hunk 1 above)
Comment (jkgoodrich) [reply gnomad_qc/v4/subset.py:166]: Yeah, let's make this another PR, this subset script will need more thought anyway for subsets that will have the final annotations also added on.
